app/src/main/res/values/strings.xml

```diff
@@ -63,13 +63,13 @@
     <string name="exporting">Exporting…</string>
     <string name="storageReadPermissionRequired">Permission to read storage needed for this action…</string>
     <string name="cameraPermissionRequired">Permission to access camera needed for this action…</string>
-    <string name="permissionReadCardsLabel">Read Catima Cards</string>
-    <string name="permissionReadCardsDescription">read your Catima cards and all its details, including notes and images</string>
-    <string name="cameraPermissionDeniedTitle">Could not access the camera</string>
+    <string name="permissionReadCardsLabel">Read Catima cards</string>
+    <string name="permissionReadCardsDescription">Read your Catima cards and all its details, including notes and images</string>
```

Comment:
This definitely needs to stay lowercase

<img width="1116" height="1097" alt="image" src="https://github.com/user-attachments/assets/5890b115-7575-409f-88b6-88061d3ba871" />